src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -74,6 +83,10 @@ def __init__(self, data, is_new, tracker):
     ccs = data['issueState'].get('ccs', [])
     self._ccs = issue_tracker.LabelStore(
         [user['emailAddress'] for user in ccs if 'emailAddress' in user])
+    collaborators = data['issueState'].get('collaborators', [])
+    self._ext_collaborators = issue_tracker.LabelStore([
```

Comment:
Is ext external or extension? Can you write out the full name.

Reply:
ext (extension) came from this comment- https://github.com/google/clusterfuzz/pull/3432#issuecomment-1769933412

I added a comment here to make this more clear.